next/components/styleguide/showcases/SnackbarShowCase.tsx

```diff
@@ -0,0 +1,119 @@
+import { useState } from 'react'
+import { useSnackbar } from 'react-simple-snackbar'
+
+import { AlertType } from '../../forms/info-components/Alert'
+import Button from '../../forms/simple-components/Button'
+import { Stack } from '../Stack'
+import { Wrapper } from '../Wrapper'
+
+const SnackbarShowCase = () => {
+  const [type, setType] = useState<AlertType>()
+
+  // This example doesn't work
+  const optionsDoesntWork = {
+    style: {
+      backgroundColor:
+        type === 'success'
+          ? 'rgb(var(--color-success-700))'
+          : type === 'info'
+          ? 'rgb(var(--color-gray-700))'
+          : type === 'warning'
+          ? 'rgb(var(--color-warning-700))'
+          : 'rgb(var(--color-negative-700))',
+    },
+  }
```

Comment:
this doesn't work because that's not how setState works, you can't set state and then read it immediately right afterwards

please remove